akka-actor/src/main/scala/akka/actor/TypedActor.scala

```diff
@@ -120,8 +125,13 @@ object TypedActor extends ExtensionId[TypedActorExtension] with ExtensionIdProvi
    *
    * Java API
    */
+  @silent
   def get(context: ActorContext): TypedActorFactory = apply(context)
 
+  @deprecated("Use 'akka.actor.typed' API.", since = "2.6.0")
+  @silent
+  override def apply(system: ActorSystem): TypedActorExtension = super.apply(system)
```

Comment:
I added this so that in the IDE a user sees the deprecation in usage, anything doing `TypedActor(..)`, if that is desired.

Reply:
Doesn't the IDE show deprecated if the only the object is deprecated?

Reply:
@johanandren Oddly for me it did not :( I tried a few times to check myself

Reply:
Ok, thanks for verifying, surprising.